outputs.tf

```diff
@@ -3,6 +3,16 @@ output "datadog_monitor_names" {
   description = "Names of the created Datadog monitors"
 }
 
+output "datadog_monitor_ids" {
+  value       = values(datadog_monitor.default)[*].name
```

Comment:
these are the same as `datadog_monitor_names`.
Should it be `values(datadog_monitor.default)[*].id`?

Reply:
ah yes, copy pasta fail :) Fixed.